src/main/java/com/continuum/nova/utils/Utils.java

```diff
@@ -7,699 +7,369 @@
-        locations.add(new ResourceLocation("gui/spectator_widgets"));
-        // locations.add(new ResourceLocation("gui/stream_indicator")); // non-square texture, breaks
-        locations.add(new ResourceLocation("gui/widgets"));
-        locations.add(new ResourceLocation("gui/world_selection"));
-        locations.add(new ResourceLocation("gui/achievement/achievement_background"));
-        locations.add(new ResourceLocation("gui/achievement/achievement_icons"));
-        locations.add(new ResourceLocation("gui/container/anvil"));
-        locations.add(new ResourceLocation("gui/container/beacon"));
-        locations.add(new ResourceLocation("gui/container/brewing_stand"));
-        locations.add(new ResourceLocation("gui/container/crafting_table"));
-        locations.add(new ResourceLocation("gui/container/dispenser"));
-        locations.add(new ResourceLocation("gui/container/enchanting_table"));
-        locations.add(new ResourceLocation("gui/container/furnace"));
-        locations.add(new ResourceLocation("gui/container/generic_54"));
-        locations.add(new ResourceLocation("gui/container/hopper"));
-        locations.add(new ResourceLocation("gui/container/horse"));
-        locations.add(new ResourceLocation("gui/container/inventory"));
-        locations.add(new ResourceLocation("gui/container/stats_icons"));
-        locations.add(new ResourceLocation("gui/container/villager"));
-        locations.add(new ResourceLocation("gui/container/creative_inventory/tabs"));
-        locations.add(new ResourceLocation("gui/container/creative_inventory/tab_inventory"));
-        locations.add(new ResourceLocation("gui/container/creative_inventory/tab_items"));
-        locations.add(new ResourceLocation("gui/container/creative_inventory/tab_item_search"));
-        locations.add(new ResourceLocation("gui/presets/chaos"));
-        locations.add(new ResourceLocation("gui/presets/delight"));
-        locations.add(new ResourceLocation("gui/presets/drought"));
-        locations.add(new ResourceLocation("gui/presets/isles"));
-        locations.add(new ResourceLocation("gui/presets/luck"));
-        locations.add(new ResourceLocation("gui/presets/madness"));
-        locations.add(new ResourceLocation("gui/presets/water"));
-        locations.add(new ResourceLocation("gui/title/minecraft"));
-        locations.add(new ResourceLocation("gui/title/mojang"));
-        locations.add(new ResourceLocation("gui/title/background/panorama_0"));
-        locations.add(new ResourceLocation("gui/title/background/panorama_1"));
-        locations.add(new ResourceLocation("gui/title/background/panorama_2"));
-        locations.add(new ResourceLocation("gui/title/background/panorama_3"));
-        locations.add(new ResourceLocation("gui/title/background/panorama_4"));
-        locations.add(new ResourceLocation("gui/title/background/panorama_5"));
+        addResources(locations,
+                "gui/bars", "gui/book", "gui/demo_background",
```

Comment:
I'm tempted to reject this cause you've changed so much spacing for no apparent reason but if you put each texture name on a different line so it's easier to read then I'll accept this bit